addons/project/controllers/portal.py

```diff
@@ -46,6 +46,7 @@ def _project_get_page_view_values(self, project, access_token, page=1, date_begi
             pager=pager,
             project=project,
             task_url=f'projects/{project.id}/task',
+            record=project,
```

Comment:
Could you adapt the existing code to use `record` instead of passing the record twice in the values?